src/routes/docs/quick-starts/nextjs/+page.markdoc

```diff
@@ -71,13 +77,14 @@ export const account = new Account(client);
 export { ID } from 'appwrite';
 ```
 {% /section %}
+
 {% section #step-5 step=5 title="Create a login page" %}
 Create or update `app/page.jsx` file and add the following code to it.
 
 ```js
 "use client";
 import { useState } from "react";
-import { account, ID } from "appwrite";
+import { account, ID } from "./appwrite";
```

Comment:
Please update only this line and the `jsx` extension, and revert all the newline and add/removed space changes in this PR to keep things concise.

Reply:
I've made the one-line change and kept the original spacing as per your request. It's important to note that the file is already in the .js format, so adding the .jsx extension is not required. Additionally, according to the Next.js documentation, explicit extensions like .js, .ts, .jsx, and .tsx are not necessary for importing files.

If you have any further questions or need additional assistance, please feel free to let me know.

Reply:
Hi, this line:
```
Create or update `app/page.jsx` file and add the following code to it.
```

is using the wrong file name compared to what's generated in Next.js. Please also fix this quickly, thanks!

Reply:
I misunderstood because I usually create Next.js apps with TypeScript, where the default extension is .tsx. I mistakenly assumed that in plain JavaScript, it would be .jsx. My apologies for the confusion. I've made the necessary change to page.js as you requested. Please review the updated code. If there are any further adjustments needed, please let me know.

Reply:
No problem